ADOPTERS.md

```diff
@@ -0,0 +1,7 @@
+# Adopters
+
+_If you're using OpenMeter in your organization, please try to add your company name to this list. It really helps the project to gain momentum and credibility. It's a small contribution back to the project with a big impact._
+
+| Organization | Contact | Description of Use |
+| ------------ | ------- | ------------------ |
+|              |         |                    |
```

Comment:
A list is probably easier to maintain than a table: https://github.com/bank-vaults/bank-vaults/blob/main/ADOPTERS.md

Also, I'm not sure Contact should be necessary.